docs/toc.md

```diff
@@ -10,6 +9,33 @@
 ### [🔧 Development using vim](getting-started/devenv/using-vim.md)
 ### [🔧 Development using Sublime Text](getting-started/devenv/using-sublime.md)
 ## [Writing your first console application on .NET Core](getting-started/first-console-app.md)
+# [Scenarios](scenarios/index.md)
+## [Writing console applications](scenarios/console/index.md)
+### [🔧 Overview and moving pieces](scenarios/console/overview.md)
+### [🔧 Console applications paradigm](scenarios/console/paradigm.md)
```

Comment:
@richlander Kewl, makes sense. This is just moved what we currently have. I can re-arrange any way we see/feel fit.